test/testSend.js

```diff
@@ -38,6 +39,15 @@ describe('bigquery tests', () => {
     assert.equal(results.length, 10);
   }).timeout(5000);
 
+  it('runs a query with queryParams', async () => {
+    const { results } = await execute(env.email, env.key, env.projectid, 'next-resource', '0bxMEaYAJV6SoqFlbZ2n1f', {
+      limit: 1,
+      tablename: 'requests201905',
+    });
+    assert.ok(Array.isArray(results));
+    assert.equal(results.length, 1);
+  }).timeout(1000000);
```

Comment:
Having to tweak the timeout right from the beginning is usually a sign of erroneous practice: do you mock the server response ? If yes, then the default timeout should be fine. If no, it means you send the request to a real server. If this is the case, I would separate those IT tests into a separate folder to clearly distinguish between: 
- the unit tests that can be run quickly on a local dev machine for each single change he makes
- the IT tests that can take longer and would run only when the dev work is finished to validate that the code he wrote works in the integrated environment.

Reply:
Yes! I have an issue up about just mocking the server response using either proxyquire or nocks

https://github.com/adobe/helix-run-query/issues/30